.github/workflows/sycl-nightly.yml

```
@@ -23,6 +23,7 @@ jobs:
       # prefer widespread gzip compression.
       artifact_archive_name: sycl_linux.tar.gz
 
+  # Build used for performance testing only: not intended for testing
```

```suggestion
  # Shared lib build is used for performance benchmarking that's why building with --no-assertions.
```
Something like that.